src/main/java/cohen/mvc/FreebiePostsFrame.java

```diff
@@ -22,7 +28,8 @@ public class FreebiePostsFrame extends JFrame
     public FreebiePostsFrame(FreebiePostsController controller,
                              @Named("postTitles") JList<String> postTitles,
```

Comment:
Why is this a constructor parameter?

Reply:
So that the menu of titles on the side can be easily updated whenever the Observable<> changes

Reply:
Do I need to change this? I tried changing it to just be a method called from the controller when anything gets updated, but I am now having an error where I cannot determine the cause, but I know it crashes on the first line where this method is called.